index.html

```diff
@@ -554,21 +554,19 @@ <h3>Characters &amp; encoding</h3>
 <section id="h_fonts">
 <h3>Fonts</h3>
 
-<p>Arabic script counts 26 letters, and mostly 19 basic shapes. Since letters change according to their position in the word, Arabic set of glyph may range to more than one hundred shapes. If one count possible ligatures, and different combination of joining forms (see above), the number of glyph can increase further. Not sure that typeface design can accommodate all needs, even though some present typefaces can run hundred of shapes.</p>
+<p>Arabic script counts 26 letters, though mostly 19 basic shapes. Since letters change according to their position in the word, Arabic  glyph sets may range to more than one hundred shapes. If one counts possible ligatures, and different combinations of joining forms (see above), the number of glyphs can increase further. It is not a given that a typeface design can accommodate all needs, even though some current typefaces can run to hundred of shapes.</p>
+<p>Early typefaces, some still in use today, incorporated design features based on a variety of simplifications. For example, one of the first approaches  used a "typewriter" style, where the same glyph is used for different positions in a word. This is the case for initial and medial shapes for most  letters. It is generally the browser default font for the Arabic script. A more unifying approach is the use of a single and detached glyphs for each letter without joining. Other approaches were used, producing visual results of more or less practicality.</p>
 
-<p>Early typefaces, some still in use today, were designed with some facilities. Designer of those differs in their simplification hypothesis. For example, one of the first approach is to use "type writer style", that is a same glyph for different positions in a word. This is the case for initial and medial shape for most of the letters (example here). It is generally the browser default font for Arabic script. A more unifying approach is the use of a single and detached glyph for each letter without joining (todo example here). Other approach were used resulting in more or less visually practical fonts.</p>
-
-<p>Nowadays, there is a large choice of fonts, and one can choose the font that best suits to one's typographical desire. However, one may also wish to take into account some non typographical considerations like: (TBD later on)</p>
+<p>Nowadays, there is a large choice of fonts, and one can choose the font that best suits  one's typographical desire. However, one may also wish to take into account some non-typographical considerations like the following:</p>
 
 <ul>
-<li>Accessibility (readability and visibility) ...</li>
-<li>The kind of device with small screen (for example, larger loop and teeth height,
-small descenders etc...), although fonts actually appear better on smartphones</li>
+<li>Accessibility (readability and visibility).</li>
+<li>Devices witha small screen (for example, larger loop and teeth height,
```

Comment:
s/witha/with a/ ? @r12a

Reply:
Yes. I'll get it next time around. There are probably still a number of edits that are needed, but i did what i could in the time available.